### PR TITLE
redirect login and get user info

### PR DESCRIPTION
**Pull request by nghiahn-0358**
[Ticket #69078](https://edu-redmine.sun-asterisk.vn/issues/69078)

### CODE REVIEW

**Comment by nghiahn-0358** [comment]
ready

**Comment by nghiahn-0358** [comment]
ready

File: src/main/java/com/example/naitei19javaecommerce/controller/user/BaseController.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    public  static CustomUserDetails loadCurrentUser (){
+        Authentication auth = SecurityContextHolder.getContext().getAuthentication();
+
+        if (!(auth instanceof AnonymousAuthenticationToken) && auth.isAuthenticated()) {
+            CustomUserDetails customUserDetails = (CustomUserDetails) auth.getPrincipal();
+            return customUserDetails;
+        }
+        else {
+            return null;
+        }
```
Comment (vanvtt-0952) [inline src/main/java/com/example/naitei19javaecommerce/controller/user/BaseController.java:18]:
```suggestion
        return null;
```

**Comment by vanvtt-0952** [comment]
merged
Comment nhỏ, fix sau gái nha

**Pull request merged**